docs/technical_reference/tools/parameter_sweep.rst

```diff
@@ -12,15 +12,28 @@ Overview
 The parameter sweep tool systematically fixes variables or modifies
 mutable parameters on a flowsheet (or Pyomo ConcreteModel), optimizes
 the flowsheet, and reports user-specified results.
-
-For each item the user wants to change, they specify a "short" name
-as a key, and then a tuple specifying the Pyomo object to be changed,
-a lower limit, an upper limit, and the number of elements to be sampled 
-for this parameter between the lower limit and upper limit.
+The parameter sweep tool can be operated in one of two ways: sweeping a 
+fixed set of paramters, or allowing for random samples from a distribution.
+While different fixed or random sampling types can be combined, in a single
+parameter sweep the user must use either all fixed or all random sampling types.
+(This implementation detail may be relaxed in future releases.)
+
+For the "fixed" sampling type, `LinearSample`, the parameter sweep tool will evaluate the 
+cross product of all the specified parameters, whereas with the "random" sampling
+types `UniformSample` and `NormalSample` the parameter sweep tool will evaluate
```

Comment:
@bknueven Is random sampling something we want to mention here, or should it be reserved for the subsequent how-to on parameter sweep that we discussed in #166? 

Either way, this shouldn't hold up the PR. If anything, a subsequent PR that adds a follow-up how-to for monte carlo sampling could describe random sampling types, and any undesired redundancies between the 2 how-to docs could then be eliminated.

Reply:
At the time I wrote this, #166 did not exist. I think it's fine for now to keep this an add on to #166 or further elaborate in subsequent documentation. At least with this the monte carlo stuff is at least mentioned.